relay-server/src/metrics_extraction/sessions.rs

```diff
@@ -10,7 +10,8 @@ use super::utils::with_tag;
 const METRIC_NAMESPACE: MetricNamespace = MetricNamespace::Sessions;
 
 /// Current version of metrics extraction.
-const EXTRACT_VERSION: u16 = 1;
+const EXTRACT_VERSION: u16 = 2;
```

Comment:
We built this version mechanism mainly to disable metrics extraction in external relays in case there is a bug in their version. For feature flagging, I would prefer if we could pass the feature flag explicitly by

1. adding the feature to [`EXPOSABLE_FEATURES`](https://cs.github.com/getsentry/sentry/blob/75a049207fb9ab51ce422c8f18bb025226c91ed8/src/sentry/relay/config/__init__.py#L46) on the sentry side,
2. adding the feature flag in Relay and passing it into metrics extraction: https://github.com/getsentry/relay/blob/1edaf631c61968066bd2f905a0b8dcfa22dc7558/relay-server/src/actors/project.rs#L71

Reply:
Discussed this with @jan-auer: we can go with the incremented EXTRACT_VERSION as you implemented it. As long as we do a slow rollout using sample_modulo on the sentry side, the increase in traffic to our own Relays should be manageable.